docs/pipelines.md

```diff
@@ -869,7 +869,7 @@ what kind of events are triggered based on the `Pipelinerun` status.
 
 ### Using Execution `Status` of `pipelineTask`
 
-Finally Task can utilize execution status of any of the `pipelineTasks` under `tasks` section using param:
+A `finally` Task can utilize execution status of the `pipelineTasks` under `tasks` section using param:
```

Comment:
NIT: maybe rephrasing this from a pipeline POV would make the sentence more clear? 
E.g. A pipeline can inject the status of specific `pipelineTasks` to tasks in the `finally` section through the task parameters.

Reply:
:) great suggestion, updated the content 👍